.repo-metadata.json

```
@@ -1,11 +1,12 @@
 {
-  "name": "pybigquery",
+  "name": "sqlalchemy-bigquery",
   "name_pretty": "SQLAlchemy dialect for BigQuery",
-  "client_documentation": "https://github.com/googleapis/pybigquery",
+  "client_documentation":
+    "https://googleapis.dev/python/sqlalchemy-bigquery/latest/index.html",
   "release_level": "beta",
   "language": "python",
   "library_type": "INTEGRATION",
   "repo": "googleapis/python-bigquery-sqlalchemy",
-  "distribution_name": "pybigquery",
+  "distribution_name": "sqlalchemy-bigquery"
```

Need extra comma now that we added `api_id`

```suggestion
  "distribution_name": "sqlalchemy-bigquery",
```

---

Fixed